packages/bp/src/orchestrator/messaging-server.ts

```diff
@@ -43,7 +44,11 @@ export const startLocalMessagingServer = (message: Partial<MessagingServerOption
   process.send!({ type: MessageType.StartMessagingServer, ...message })
 }
 
-export const startMessagingServer = async (opts: Partial<MessagingServerOptions>, logger: sdk.Logger) => {
+export const startMessagingServer = async (
+  logger: sdk.Logger,
+  opts?: Partial<MessagingServerOptions>,
```

Comment:
Why put this optional? I'm pretty sure, that having a proper EXTERNAL_URL and SPINNED_URL could be something useful when running migrations in the future.

Reply:
I'm not sure it's something that should be required. Each of those processes are made to be executed standalone, they should be able to do their migrations without that information. If it is ever required eventually, let's adjust it, but I prefer that we keep those processes decoupled